packages/components/src/components/Dropdown/styles/dropdown.module.scss

```diff
@@ -53,6 +49,10 @@
         order: 2;
         margin-left: auto;
     }
+
+    &:not(&:hover) {
```

Comment:
I think this is not valid but once you remove that second `&` you might end up with the focus outline styling applying to the element any time it isn't hovered. I think this declaration might not be necessary since currently it's probably not doing anything

Reply:
This might do it

```suggestion
    &:hover:not(:focus-visible) {
        outline: none;
    }
```

Reply:
fixed

Reply:
The suggestion of José isn't applied, it would fix the issue with the mouse being on top of an element not adding the outline when switching to keyboard

Reply:
No it wouldn't because the elements are indeed "focus-visible" once the mouse hovers over them.
So this rule would not do anything.

Reply:
yep, you're right. been reading a bit more on this and it seems like [it was quite contentious](https://github.com/radix-ui/primitives/issues/1164) but apparently inline with a11y patterns to have the focus be the same as the hover visually. Then they introduced the [data-highlighted](https://github.com/radix-ui/primitives/issues/1289) to allow devs to get around this limitation but it can't be done without the hover excluding the outline I don't think